_data/istio.yml

```diff
@@ -1 +1,2 @@
 version: 0.3.0
```

Comment:
change to 0.4.0 while touching this file

Reply:
Done